src/integrant/core.clj

```diff
@@ -48,12 +48,37 @@
 (defn- sort-keys [ks m]
   (sort (dep/topo-comparator (dependency-graph m)) ks))
 
+(defn- update-key [f m k]
```

Comment:
Change the argument order to `[m k f]` instead, to match the argument order of `clojure.core/update`.